src/EReg/Util.h

```diff
@@ -1,9 +1,27 @@
 #pragma once
 
+#include <PIDController.h>
+
 namespace Util {
+
+    #define MAX_SPD 255
+    #define MIN_SPD -255
+    // #define STATIC_SPD 60
+    #define STATIC_SPD 0
+    //old max angle = 1296
+    //3200*48/26
+    #define MAX_ANGLE 363
+    #define MIN_ANGLE 0
```

Comment:
Put these (incl PID constants) in a separate config.cpp / constants.cpp file so its easier to version control?